Documentation/faq.md

```diff
@@ -62,6 +62,22 @@ With longer latencies, the default etcd configuration may cause frequent electio
 
 etcdctl provides a `snapshot` command to create backups. See [backup][backup] for more details.
 
+#### Always remove first when replacing member?
```

Comment:
Also i wonder if it is a good idea to link related issue with the question in faq. in that way, it gives reader a context. what do you think?